config/alphagov.yml

```diff
@@ -127,6 +127,30 @@ govuk-platform-health:
     - "[DO NOT MERGE]"
     - "WIP"
 
+ quotes:
+    - Continue getting involved in Hong's weird team activities
+    - If a fika isn't happening, start it
+    - Fika is a work-free environment
+    - We encourage you to do Fika everyday - don't be shy about initiating it
+    - Team lunch anyone?
+    - Listen to each other's points of view
+    - Spot opportunities, be brave in announcing them
+    - Ask for more clarity if needed
+    - Feel free to ask for help/questions/etc
+    - Try pairing, as well as not pairing...both are good
+    - Ask questions when you don't know something - it will also give others the confidence to ask their questions
+    - It's OK to not know how something works
+    - Be supportive - knowledge sharing helps the team achieve its goals
+    - Don't rush on to the next thing, pause, help out, deliver work as a team
+    - Work at a considered / sustainable pace 
+    - Take responsibility for your work until it's deployed
+    - Awareness of everyone's preferences on ways of working (more info: https://drive.google.com/drive/folders/1qllr2HfwGg6JL1VCWPCQusMokYYCq_-Z)
```

Comment:
You need to put this one around quote marks (") because it has a colon in it.

Reply:
`- "Awareness of everyone's preferences on ways of working (more info: https://drive.google.com/drive/folders/1qllr2HfwGg6JL1VCWPCQusMokYYCq_-Z)"`

Reply:
Fanks Tom